CRM/ManualDirectDebit/Upgrader.php

```diff
@@ -0,0 +1,40 @@
+<?php
+use CRM_ManualDirectDebit_ExtensionUtil as E;
+
+/**
+ * Collection of upgrade steps.
+ */
+class CRM_ManualDirectDebit_Upgrader extends CRM_ManualDirectDebit_Upgrader_Base {
+
+  // By convention, functions that look like "function upgrade_NNNN()" are
+  // upgrade tasks. They are executed in order (like Drupal's hook_update_N).
+
+  /**
+   * Example: Run an external SQL script when the module is installed.
```

Comment:
please remove these auto-generated docblocks from all this upgrader class methods.

Reply:
don